build.gradle

```diff
@@ -100,6 +101,10 @@ configurations {
     integrationTestRuntimeOnly.extendsFrom runtimeOnly
     tdvtTestImplementation.extendsFrom implementation
     tdvtTestRuntimeOnly.extendsFrom runtimeOnly
+
+    // Use the release version instead of gitVersion for the produced jar
+    jar.archiveVersion = releaseVersion
+    shadowJar.archiveVersion = releaseVersion
```

Comment:
I think it was brought up in the meeting, did we decide on whether we wanted to add more of the git version to the produced jar file to know which checkin it was built from?

Reply:
From my notes, having the "full" version in the manifest was Ok and enough for us to be able to track which commit the snapshot was linked to. If I misunderstood, I can change releaseVersion to gitVersion first thing tomorrow.

Reply:
Ah that's what it was.  Yes I see the manifest has the version with the git hash in it. 👍